src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -6,6 +6,11 @@ CA1418 | Interoperability | Warning | UseValidPlatformString, [Documentation](ht
 CA1839 | Performance | Info | UseEnvironmentMembers, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1839)
 CA1840 | Performance | Info | UseEnvironmentMembers, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1840)
 
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA1841 | Performance | Info | PreferDictionaryContainsMethods, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1839)
+
```

Comment:
```suggestion
CA1841 | Performance | Info | PreferDictionaryContainsMethods, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1839)

```

Reply:
Wait, shouldn't the doc link be changed to ca1841?

Reply:
Yes, correct

Reply:
fixed.